system_tests/local/ec2_test_utils.py

```diff
@@ -224,3 +284,9 @@ def _create_instance(self, ec2_client):
             image_id=self.env.ubuntu_trusty_image_id,
             instance_type=self.env.micro_instance_type)
         return new_reservation.instances[0]
+
+    def _create_external_load_balancer(self):
+        return self._get_elb_client().create_load_balancer(
```

Comment:
this is an external method which uses boto directly. or did I misunderstand your original intention ?